scripts/cmake-presets/jlse.json

```diff
@@ -37,13 +37,13 @@
     {
       "name": "release",
       "displayName": "Build with optimizations and disable debug assertions",
-      "inherits": [".base", ".release"],
+      "inherits": [".base", ".ndebug"],
       "binaryDir": "${sourceDir}/build-release"
     },
     {
       "name": "release-amd",
       "displayName": "Release build for an AMD device",
-      "inherits": [".base", ".release-amd"],
```

Comment:
@elliottbiondo did you define this one locally? I couldn't find it in the base or jlse cmake presets.

Reply:
No, these were mistakes on my part, thanks for fixing!

Just to clarify, you aren't using the `build.sh` script for these, but soft-linking to `CMakePresets.json` and building via `cmake`?

Reply:
Correct!